patches/src/main/kotlin/app/revanced/patches/angulus/misc/license/DisableLicenseCheckPatch.kt

```diff
@@ -0,0 +1,21 @@
+package app.revanced.patches.angulus.misc.license
+
+import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.returnEarly
+
+@Suppress("unused")
+val disableLicenseCheckPatch = bytecodePatch(
+    name = "Disable license check",
+    description = "Disable client-side license check to prevent app from closing on startup."
+) {
+    compatibleWith("com.drinkplusplus.angulus"("5.0.20"))
```

Comment:
```suggestion
    compatibleWith("com.drinkplusplus.angulus")
```

Considering this patch is more specific to pairip than Angulus, it's unlikely to change